_includes/pbc/all/install-features/202407.0/install-the-cart.md

```diff
@@ -0,0 +1,176 @@
+
+
+{% info_block infoBox %}
+
+This document describes how to add product variants, product images, and price validation for a cart's items to an existing cart.
+
+{% endinfo_block %}
+
+## Prerequisites
+
+Learn about [super attributes](/docs/pbc/all/product-information-management/{{page.version}}/base-shop/feature-overviews/product-feature-overview/product-attributes-overview.html#super-attributes).
+
+## UI changes
+
+Cart items can now be changed by changing their attributes. For example, you can change a t-shirt's size.
+
+Cart supports product images out of the box.
+
+![cart_product_images](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/dev/feature-integration-guides/cart-integration.md/cart_product_images.png)
+
+Based on the super attributes, you can select the needed product variant in the cart.
+
+![product_super_attributes](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/dev/feature-integration-guides/cart-integration.md/product_super_attributes.png)
+
+## Add item images
+
+1. To support images to cart, install the optional module `ProductImageCartConnector`:
+
+```bash
+composer require spryker/product-image-cart-connector
+```
+
+2. This module provides `ProductImageCartPlugin`, which you must register later in your shop's `CartDependencyProvider` as follows:
+
+```php
+/**
+ * @param \Spryker\Zed\Kernel\Container $container
+ *
+ * @return \Spryker\Zed\Cart\Dependency\ItemExpanderPluginInterface[]
+ */
+protected function getExpanderPlugins(Container $container)
+{
+    return [
+        // your existing plugins ...
+        new ProductImageCartPlugin(),
+    ];
+}
+```
+
+If your shop uses product bundles, register `ExpandBundleItemsWithImagesPlugin` in your shop's `CartDependencyProvider` as follows:
+
+```php
+/**
+ * @param \Spryker\Zed\Kernel\Container $container
+ *
+ * @return \Spryker\Zed\Cart\Dependency\ItemExpanderPluginInterface[]
+ */
+protected function getExpanderPlugins(Container $container)
+{
+    return [
+	  //your existing plugins
+        new ExpandBundleItemsWithImagesPlugin(),
+        ];
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure `ExpandBundleItemsWithImagesPlugin` is registered after the `ExpandBundleItemsPlugin` plugin.
```

Comment:
How do I verify this?

Reply:
Removed the outdated guide and added this part to the bundle product feature in IG.